index.d.ts

```diff
@@ -1,12 +1,65 @@
-import { Parser } from 'acorn' 
+import * as acorn from 'acorn';
 
-declare const jsx: (options?: jsx.Options) => (BaseParser: typeof Parser) => typeof Parser;
+declare const jsx: {
+  tokTypes: typeof acorn.tokTypes;
+  (options?: jsx.Options): (BaseParser: typeof acorn.Parser) => jsx.AcornJsxParser
+}
 
 declare namespace jsx {
-  interface Options {
+
+  type tokTypes = typeof acorn.tokTypes;
+
+  export interface TokTypes extends tokTypes {
+    jsxName: acorn.TokenType,
+    jsxText: acorn.TokenType,
+    jsxTagEnd: acorn.TokenType,
+    jsxTagStart: acorn.TokenType
+  }
+
+  export interface Options {
     allowNamespacedObjects?: boolean;
     allowNamespaces?: boolean;
   }
+
+  export type TokContexts = {
```

Comment:
Nit: I personally prefer interfaces. This is also more consistent with the rest of the code base.

Reply:
Sure. Done.